fix: don't change the variable type and filter all values in JSON request (#6954)
codeigniter4/CodeIgniter4: user_guide_src/source/changelogs/v4.3.0.rst

**********

- Fixed a bug when all types of ``Prepared Queries`` were returning a ``Result`` object instead of a bool value for write-type queries.

> - Fixed a bug when not always all variables in a JSON request were filtered if filters were applied with ``IncomingRequest::getVar()`` or ``IncomingRequest::getJsonVar()`` methods.

Reviewer: ```suggestion
- Fixed a bug with variable filtering in JSON requests ysing with ``IncomingRequest::getVar()`` or ``IncomingRequest::getJsonVar()`` methods.
```

Reviewer 2: What's *ysing* ?

Author: Thanks. Fixed.

Reviewer: Haha oops! Sorry for the typo, yes that was supposed to be "using" (and drop the "with"). Looks good.
